Source/Client/Windows/ServerBrowser.cs

```diff
@@ -362,14 +385,15 @@ private void DrawSaveList(List<SaveFile> saves, float width, ref float y)
                     TooltipHandler.TipRegion(outdated, text);
                 }
 
+
                 Text.Font = GameFont.Small;
                 GUI.color = Color.white;
 
                 if (Widgets.ButtonInvisible(entryRect))
                 {
                     if (Event.current.button == 0)
                         selectedFile = saveFile;
-                    else
+                    else if(saveFile.Valid)
```

Comment:
If the save file isn't valid, clicking on it won't have any effect. Seems correct.